indico/web/client/js/widgets/lang_selector.js

```diff
@@ -0,0 +1,11 @@
+// This file is part of Indico.
+// Copyright (C) 2002 - 2025 CERN
+//
+// Indico is free software; you can redistribute it and/or
+// modify it under the terms of the MIT License; see the
+// LICENSE file for more details.
+
+// eslint-disable-next-line import/unambiguous
+window.addEventListener('DOMContentLoaded', () => {
+  document.getElementById('language-selector-inline').onclick = evt => evt.preventDefault();
```

Comment:
I'm not a fan of setting `onclick` directly instead of using `addEventListener`...

I think it would also be a good to add a comment here why this is needed.